xdsl/parser/core.py

```diff
@@ -730,7 +731,8 @@ def parse_operation(self) -> Operation:
         if (n_bound_results != 0) and (len(op.results) != n_bound_results):
             self.raise_error(
                 f"Operation has {len(op.results)} results, "
-                f"but was given {n_bound_results} to bind."
+                f"but was given {n_bound_results} to bind.",
```

Comment:
@math-fehr by the time this error gets printed the current token location points to the next operation, I got quite confused for a bit :)